unbxdSearch.js

```diff
@@ -1063,9 +1063,9 @@ var unbxdSearchInit = function(jQuery, Handlebars){
 	     && this.defaultParams.ranges.hasOwnProperty(x)
 	     && this.defaultParams.ranges[x].hasOwnProperty(y)
 	     && this.params.ranges[x].hasOwnProperty(y)){
-	    b.push(x + ':[' + this.params.ranges[x][y].lb + " TO " + this.params.ranges[x][y].ub + ']');
+	    b.push(x + ':[' + this.params.ranges[x][y].lb + " TO " + this.params.ranges[x][y].ub + '}');
 	  }else if(this.params.ranges[x].hasOwnProperty(y)){
-	    a.push(x + ':[' + this.params.ranges[x][y].lb + " TO " + this.params.ranges[x][y].ub + ']');
+	    a.push(x + ':[' + this.params.ranges[x][y].lb + " TO " + this.params.ranges[x][y].ub + '}');
```

Comment:
```
square bracket -> Include interval
flower bracket -> Exclude interval
```
Currently we are apply filter including both upper and lower Interval. But in facets response, it returns count of products under `Start >= product < End`